lib/HLSL/HLOperationLower.cpp

```diff
@@ -6068,7 +6068,53 @@ Value *TranslateHitObjectMake(CallInst *CI, IntrinsicOp IOP, OP::OpCode opcode,
                               HLOperationLowerHelper &helper,
                               HLObjectOperationLowerHelper *pObjHelper,
                               bool &Translated) {
```

Comment:
Normally I wouldn't ask you to clean up pre-existing code, but in this case you added the code and the change here modifies the whole function, so please update the parameter names and all the variables throughout the body to conform to LLVM Coding Standards (https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly)

Reply:
Reworked the variable names to follow the coding standards